### PR TITLE
Deal with duplicate judgements (and some other polish)

### PR DESCRIPTION
**Pull request by epugh**
## Description
Some polish to how we handle judgements based on hard won experience in the field...

## Types of changes

- [] Bug fix (non-breaking change which fixes an issue)
- [] Improvement (non-breaking change which improves existing functionality)
- [] New feature (non-breaking change which adds new functionality)
- [] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [] My code follows the code style of this project.
- [] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [] All new and existing tests passed.

### CODE REVIEW

**Comment by epugh** [comment]
If you get a chance @depahelix2021 can you code review this?    I think I found some fixes...

**Review by depahelix2021 — COMMENTED** [review]
Couple questions

File: app/controllers/judgements_controller.rb [hunk 1]
```diff
@@ -24,39 +23,35 @@ def new
     if @query_doc_pair
       @query = @current_user.queries.has_information_need.where(query_text: @query_doc_pair.query_text).first
     end
-    @judgement = Judgement.new
+    @judgement = Judgement.new(query_doc_pair: @query_doc_pair, user: @current_user, updated_at: Time.zone.now)
   end
 
-  def edit; end
+  def edit
```
[the remark this answers is not in the document]
Comment (epugh) [reply app/controllers/judgements_controller.rb:29]: I think so...   You can do either formatting style, so we could go back to `def edit; end`

**Review by depahelix2021 — APPROVED** [review]
looks good

**Pull request merged**